.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+## What are the changes introduced in this PR?
+
+Write a brief explainer on your code changes.
+
+## Please explain the objectives of your changes below:
+
+Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here
+
+### Type of change
+
+If the pull request is a **bug-fix**, **enhancement** or a **refactor**, please fill in the details on the changes made.
+| Current changes | New changes |
+|--|--|
+| | |
```

Comment:
What do we expect to fill in here?

Reply:
Ok. It might not be easy to do this within a table. I'd just plainly describe them one after another.

Reply:
Are the words current changes and new changes fine to use ?

Reply:
Hmm...let's define sections like "Existing capabilities/behavior" and "New behavior/capabilities".

Reply:
Updated.